src/compas/robots/model/robot.py

```diff
@@ -59,12 +58,12 @@ class RobotModel(Data):
 
     """
 
-    def __init__(self, name, joints=[], links=[], materials=[], **kwargs):
+    def __init__(self, name=None, joints=(), links=(), materials=(), **kwargs):
```

Comment:
i think these can all be `None`

Reply:
These are actually magical. I don't know where to document this, but we rely on the default data types to do some introspection and determine the expected types during urdf parsing, so, I need these to be a sequence-type, but I changed from `[]` to `()` to avoid the problem of empty mutable lists in default params.